### PR TITLE
fix responsiveness

### PR DESCRIPTION
**Pull request by bhagirathpaliyal**
Screenshot 2024-07-30 161324
Screenshot 2024-07-30 161524
Screenshot 2024-07-30 161619

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -170,18 +171,18 @@ function App() {
...
-            <div className="w-full flex flex-row justify-between">
+            <div className="w-full flex flex-row max-sm:flex-col justify-between">
               <FiltersSelector
                 benchmarks={benchmarks}
                 filters={filter}
                 setFilters={setFilter}
               />
 
-              <div className="space-x-4">
+              <div className="flex gap-[10px] max-sm:flex-col max-lg:flex-col pb-4">
```
Comment (yogeshpaliyal) [inline src/App.tsx]: Keep gap as 8px instead of 10px

File: src/App.tsx [hunk 2]
```diff
@@ -203,13 +208,18 @@ function App() {
...
 
-            <Tabs defaultValue="charts" className="w-full">
+            <Tabs defaultValue="charts">
               <TabsList className="grid w-full grid-cols-2">
                 <TabsTrigger value="charts">Charts</TabsTrigger>
                 <TabsTrigger value="table">Table</TabsTrigger>
               </TabsList>
               <TabsContent value="charts">
-                <CompareChart filter={filter} benchmarks={filteredBenchmarks} />
+                <div className=" ">
```
Comment (yogeshpaliyal) [inline src/App.tsx:217]: Remove className

**Review by yogeshpaliyal — CHANGES_REQUESTED** [review]

File: src/ui/Filters.tsx [hunk 3]
```diff
@@ -48,10 +48,10 @@ export function FiltersSelector({
   }, [benchmarks]);
 
   return (
-    <div className="space-x-4 pb-4">
+    <div className="flex gap-[10px] max-sm:flex-col max-lg:flex-col pb-4 ">
```
[the remark this answers is not in the document]
Comment (bhagirathpaliyal) [reply src/ui/Filters.tsx]: done

File: src/ui/Compare.tsx [hunk 4]
```diff
@@ -142,7 +142,7 @@ function BenchmarkSelector({
       {!!selectedBenchmark ? <h6>{label}</h6> : ""}
       <DropdownMenu>
         <DropdownMenuTrigger asChild>
-          <Button variant="outline">
+          <Button  variant="outline" >
```
[the remark this answers is not in the document]
Comment (bhagirathpaliyal) [reply src/ui/Compare.tsx]: done

File: src/ui/Filters.tsx [hunk 5]
```diff
@@ -48,10 +48,10 @@ export function FiltersSelector({
...
       <DropdownMenu>
         <DropdownMenuTrigger asChild>
-          <Button variant="outline">Select Benchmarks</Button>
+          <Button   variant="outline">Select Benchmarks</Button>
```
[the remark this answers is not in the document]
Comment (bhagirathpaliyal) [reply src/ui/Filters.tsx]: done

File: src/App.tsx (hunk 2 above)
Comment (bhagirathpaliyal) [reply src/App.tsx]: done

File: src/App.tsx (hunk 1 above)
Comment (bhagirathpaliyal) [reply src/App.tsx]: done

**Review by yogeshpaliyal — APPROVED** [review]

**Pull request merged**